src/main/java/com/hms_networks/americas/sc/extensions/system/application/SCAppManagement.java

```diff
@@ -21,6 +21,17 @@ public class SCAppManagement {
   /** Seconds to pause thread when waiting for WAN IP */
   private static final int SECONDS_IN_WAIT_PERIOD = 20;
 
+  /**
```

Comment:
This is more personal preference and kind of a more c oriented behavior of mine. I prefer all the private methods and field at the top of the file.